src/Akka.Persistence.Sql/persistence.conf

```diff
@@ -48,7 +48,7 @@
       # Linq2Db by default will use a built string for multi-row inserts
       # Somewhat counterintuitively, this is faster than using parameters in most cases,
       # But if you would prefer parameters, you can set this to true.
-      prefer-parameters-on-multirow-insert = true
+      prefer-parameters-on-multirow-insert = false
```

Comment:
Change `prefer-parameters-on-multirow-insert` to false.
Use of better optimized path should be encouraged.